tests/ft_sh.vader

```diff
@@ -12,3 +12,6 @@ Execute (shellcheck):
   set ft=sh
   norm i#!/usr/bin/env bash
   AssertEqual neomake#makers#ft#sh#shellcheck()['args'], ['-fgcc']
+
+  norm ggcG# shellcheck shell=bash
```

Comment:
I was just following the current style in the file, which is my general approach to code I do not own. I'm happy to change the code, though it seems inconsistent with the rest of the file. It seems to be littered with them.

Would you want me to change the other instances?